.stylelintrc.json

```diff
@@ -1,3 +1,17 @@
 {
-  "extends": "stylelint-config-standard"
+  "extends": "stylelint-config-standard",
+  "plugins": [
+    "stylelint-scss"
+  ],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "function-calc-no-invalid": null,
```

Comment:
Needed due to:
- https://github.com/stylelint/stylelint/issues/4114
- https://github.com/stylelint/stylelint/issues/4474